.env.example

```
@@ -0,0 +1,46 @@
+CORS_ORIGIN=http://localhost:5173
+PROJECT_STATE_PATH=.dev/lib/project-state.js
+AIDESIGNER_PROJECT_ROOT=
+
+## UI Integration Limits
+UI_INTEGRATION_MAX_ARGS=25
+UI_INTEGRATION_MAX_OUTPUT_BYTES=10485760
+UI_INTEGRATION_TIMEOUT_MS=300000
+
+## MCP Integrations
+SHADCN_MCP_COMMAND=npx
+SHADCN_MCP_ARGS=aidesigner mcp shadcn
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SHADCN_MCP_ARGS key should go before the SHADCN_MCP_COMMAND key

(UnorderedKey)